sdk/highlight-next/src/server.edge.ts

```diff
@@ -5,6 +5,7 @@ import * as withHighlightEdge from './util/with-highlight-edge'
 import type { HighlightEnv } from './util/types'
 export { H } from './util/highlight-edge'
 export { highlightMiddleware } from './util/highlight-middleware'
+export { isNodeJsRuntime } from './util/is-node-js-runtime'
```

Comment:
is this used here?

Reply:
It's only exported for compatibility. The `packages.json` `exports` attribute switches between `server.js` and `server.edge.js`, so TS expects this function to exist in both.